tests/manual-test-cases/Group5-Functional-Tests/5-14-Remove-Container-OOB.robot

```diff
@@ -24,10 +24,11 @@ Docker run an image from a container that was removed OOB
 
     Install VIC Appliance To Test Server
 
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    ${rc}  ${out}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
     Should Be Equal As Integers  ${rc}  0
     ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -itd --name removeOOB busybox /bin/top
     Should Be Equal As Integers  ${rc}  0
-    Destroy VM OOB  removeOOB*
-    ${rc}  ${container}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -itd busybox /bin/top
-    Should Be Equal As Integers  ${rc}  0
+
+    ${rc}  ${out}=  Run And Return Rc And Output  govc vm.destroy %{VCH-NAME}/removeOOB*
+    Should Not Be Equal As Integers  ${rc}  0
```

Comment:
Are we sure that the ${rc} is 0 in this case?

Reply:
should not be equal

Reply:
Sorry my bad. I was probably reading the deleted lines :)